boards/renesas/ek_ra8d1/doc/index.rst

```diff
@@ -110,6 +110,8 @@ The below features are currently supported on Zephyr OS for EK-RA8D1 board:
 +--------------+------------+------------------+
 | CAN          | on-chip    | canfd            |
 +--------------+------------+------------------+
+| ADC          | on-chip    | adc              |
```

Comment:
Wrong commmit message. This shoud start with `boards: renesas: ...`

Reply:
Thank you, I have changed the commit message and split this modification into a commit